internal/conductorr/api/radarr.go

```diff
@@ -206,6 +246,32 @@ func GetRadarrMovieLookup(w http.ResponseWriter, r *http.Request) {
 		RespondRaw(w, r, err, nil)
 		return
 	}
+
+	// Get media that already exists
+	tmdbIDs := make([]int, 0, len(results.Results))
+	for _, result := range results.Results {
+		split := strings.Split(result.ID, ":")
+		if len(split) != 2 {
+			continue
+		}
+		id, err := strconv.Atoi(split[1])
+		if err != nil {
+			continue
+		}
+		tmdbIDs = append(tmdbIDs, id)
+	}
+	existingMedias, err := dbstore.GetMediaByTmdbIDs(tmdbIDs)
+	if err != nil {
+		RespondRaw(w, r, err, nil)
+		return
+	}
+	tmdbIDToMedia := make(map[int]*dbstore.Media)
+	for _, media := range existingMedias {
+		if media.TmdbID.Valid {
+			tmdbIDToMedia[int(media.TmdbID.Int32)] = media
+		}
+	}
```

Comment:
It's kind of weird, but the way `/api/v3/movie/lookup` works is returning a `RadarrMovie/MovieResource` set with elements that contain an `id` iff the movie is already being tracked in Radarr.

and Jellyseerr relies on this behavior

https://github.com/seerr-team/seerr/blob/7d4b2853dcade1d2dc8804b400f8ebc972829a63/server/api/servarr/radarr.ts#L132